src/components/BookPage.js

```diff
@@ -0,0 +1,25 @@
+import { v4 as uuidv4 } from 'uuid';
+import Book from './book';
+import Form from './form';
+
+const BookPage = () => {
+  const books = [
+    { id: uuidv4(), title: 'The Great Gatsby', author: 'Scott Fitzgerald' },
+    { id: uuidv4(), title: 'Moby Dick', author: 'Herman Melville' },
+    { id: uuidv4(), title: 'War and Peace', author: 'Leo Tolstoy' },
+    { id: uuidv4(), title: 'The Brothers Karamazov', author: 'Fyodor Dostoyevsky' },
+  ];
+
+  return (
+    <>
+      <ul className="books">
+        {books.map((book) => (
+          <Book key={book.id} title={book.title} author={book.author} />
```

Comment:
- [ ] According to the project requirements, you should create a component that contains a list to render individual books.  I recommend creating a separate component named <BookList> specifically for this purpose. Doing so will help you meet the project requirements more effectively.👍

![book](https://github.com/Uzair-Manzoor/bookstore/assets/40009354/8010504a-64ee-4ab8-adf2-7c8700ccb383)

You could use the below as a guide:
```
// In Book.js
import React, { useState } from 'react';

const Book = ({ book, onDelete }) => {
  const [isDeleted, setIsDeleted] = useState(false);

  const handleDelete = () => {
    onDelete(book.id);
    setIsDeleted(true);
  };

  return (
    <div className={`book ${isDeleted ? 'deleted' : ''}`}>
      <h3>{book.title}</h3>
      <p>Author: {book.author}</p>
      <button onClick={handleDelete}>Delete</button>
    </div>
  );
};


```

```
// In BookList.js
import Book from './Book';

const BookList = ({ books, onDelete }) => {
  return (
    <div className="book-list">
      {books.map((book) => (
        <Book key={book.id} book={book} onDelete={onDelete} />
      ))}
    </div>
  );
};


```

```
// In BookPage.js
import BookList from './components/BookList';

function BookPage() {
  const books = [
    { id: 1, title: 'Book 1', author: 'Author 1' },
    { id: 2, title: 'Book 2', author: 'Author 2' },
    { id: 3, title: 'Book 3', author: 'Author 3' },
  ];

  const handleDeleteBook = (id) => {
    const updatedBooks = books.filter((book) => book.id !== id);
    setBooks(updatedBooks);
  };

  return (
    <>
      <BookList books={books} onDelete={handleDeleteBook} />
    </>
  );
}

```